doc/ovs-hw-offload.md

```diff
@@ -9,63 +9,148 @@ host net-device. The VF representor plays the same role as TAP devices
 in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
 arrives to the VF, and a packet sent through the VF is received by its representor.
 
+OVS Hardware Offloading requires NIC to be configured in `switchdev` mode.
+
+The operator can automate the creation and configuration of OVS bridges when the "manageSoftwareBridges" function is activated.
+
 ## Supported Ethernet controllers
 
 The following manufacturers are known to work:
 
 - Mellanox ConnectX-5 and above
 
-## Instructions for Mellanox ConnectX-5
-
 ## Prerequisites
 
 - OpenVswitch installed
-- Network Manager installed
+
+### Activate "manageSoftwareBridges" featureGate
+
+```
+kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "manageSoftwareBridges": true  } } }' --type='merge'
+```
+
 
 ### Deploy SriovNetworkNodePolicy
 
+The example policy below selects all NVIDIA ConnectX-6 Dx devices on all worker nodes.
+
+The following actions will apply to selected NICs:
+
+* set NIC's eswitch mode to `switchdev`
+
+* create 5 Virtual Functions on each Physical Function
+
+* create a separate OVS bridge (with default configuration that is suitable for HW-offloading with OVS-kernel dataplane) for each Physical Function (PF)
+
 ```yaml
 apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetworkNodePolicy
 metadata:
   name: ovs-hw-offload
   namespace: sriov-network-operator
 spec:
-  deviceType: netdevice
+  eSwitchMode: switchdev
+  mtu: 1500
   nicSelector:
-    deviceID: "1017"
-    rootDevices:
-    - 0000:02:00.0
-    - 0000:02:00.1
-    vendor: "15b3"
+    deviceID: 101d
+    vendor: 15b3
   nodeSelector:
-    feature.node.kubernetes.io/network-sriov.capable: "true"
-  numVfs: 8
-  priority: 10
-  resourceName: cx5_sriov_switchdev
-  isRdma: true
-  eSwitchMode: switchdev
-  linkType: eth
+    node-role.kubernetes.io/worker: ""
+  numVfs: 5
+  resourceName: switchdev
+  bridge:
+    ovs: {}
 ```
 
-### Create NetworkAttachmentDefinition CRD with OVS CNI config
+_Note: `spec.bridge.ovs: {}` - means use default settings (suitable for HW-offloading with OVS-kernel dataplane)
+The fields defined in the [Bridge type](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/4563178259d2f6f61451d167d3e4affdb8f2fc3a/api/v1/sriovnetworknodepolicy_types.go#L84) can be used to configure advanced bridge and interface level options._
```

Comment:
do we want the link to point to master if there are changes in the API in the future?

Reply:
done